package.json

```diff
@@ -15,9 +16,10 @@
     "storybook:preview": "NODE_ENV=production pnpm storybook --docs",
     "storybook:clean": "rm -rf storybook/public",
     "storybook:build": "storybook build -c storybook -o storybook/public",
-    "storybook:build:test": "pnpm storybook:build --test",
-    "storybook:build:prod": "pnpm storybook:build --docs",
-    "test:storybook": "test-storybook --config-dir storybook --skipTags='skip-test' --url http://127.0.0.1:6006",
+    "storybook:test": "test-storybook --config-dir storybook --skipTags='skip-test' --url http://127.0.0.1:6006",
+    "turbo:storybook:build": "pnpm turbo storybook:build -- --stats-json",
```

Comment:
The `stats-json` flag isn't needed normally for our use cases, but it is needed for Chromatic. Better to not put it in the default script as it takes longer to run